Sources/NIOCore/BSDSocketAPI.swift

```diff
@@ -191,6 +193,11 @@ extension NIOBSDSocket.AddressFamily {
 
 // Protocol Family
 extension NIOBSDSocket.ProtocolFamily {
+#if os(WASI)
+    /// UNIX local to the host.
+    public static let unix: NIOBSDSocket.ProtocolFamily =
+            NIOBSDSocket.ProtocolFamily(rawValue: 1)
```

Comment:
`PF_INET` and `PF_INET6` are available but `PF_UNIX` isn't. 1 is bound to `PF_INET`. https://github.com/WebAssembly/wasi-libc/blob/09683b3623ec49f79899fc1d847f29e26af80fd1/libc-bottom-half/headers/public/__header_sys_socket.h#L29-L30